test/integration/open-with/OpenWith.int.js

```diff
@@ -0,0 +1,22 @@
+// <reference types="Cypress" />
+
+describe('OpenWith', () => {
+    beforeEach(() => {
+        cy.server();
+    });
+
+    it('integrations will render when the dropdown is clicked', () => {
+        cy.route(
```

Comment:
If other tests are going to reuse this data, consider moving it to `before` or `beforeEach`, instead.

Reply:
Other tests will be routing with different responses. But if I write more with this data I will use a local before each to that suite.